.npmignore

```diff
@@ -14,4 +13,17 @@
 /tests
 
 /doc
-/src/parsers/manifest/dash/wasm-parser/target
+/src
+/.github
+/scripts
+
+/FILES.md
+/CONTRIBUTING.md
+/MAINTAINERS.md
```

Comment:
This one makes sense I think when published

Reply:
you mean MAINTAINERS ? or the 3 files ?

Reply:
just `MAINTAINERS.md`